cmd/lotus/config.go

```diff
@@ -31,7 +31,7 @@ var configDefaultCmd = &cli.Command{
 	Action: func(cctx *cli.Context) error {
 		c := config.DefaultFullNode()
 
-		cb, err := config.ConfigUpdate(c, nil, !cctx.Bool("no-comment"))
+		cb, err := config.ConfigUpdate(c, nil, !cctx.Bool("no-comment"), config.KeepUncommented(config.MatchEnableSplitstoreField))
```

Comment:
Let's provide a default. We shouldn't have things mentioning the splitstore all over lotus.